datadog-cloudformation-common/pom.xml

```diff
@@ -6,6 +6,13 @@
     <packaging>jar</packaging>
     <version>1.0.1</version>
     <name>datadog-cloudformation-common</name>
+    <repositories>
```

Comment:
Any reason not to reference the parent pom here too?

Reply:
The parent pom has datadog-cloudformation-common as a dependency and it's not possible to exclude it in a "child" pom, so either we:

* Keep it as it is and update the Java client dep in 2 places.
* Exclude the "common" dependency here and then maintain it in all the resources to be able to inherit the parent pom in "common".

There's no best way here, but at least this makes us only maintain 2 places.

Reply:
Ahh ok, thanks for the explanation! Seems better to maintain it in fewer places 😄